File: C++/AtCoder/ふるい/ABC180-A.cpp
// copyright
#include<bits/stdc++.h>

int solver(int n, int a, int b) {
    return n-a+b;
}

int main() {
    // input
    int n, a, b;
    std::cin >> n >> a >> b;
    int ans;
    ans = solver(n, a, b);
    std::cout << ans << std::endl;
}

File: Python/AtCoder/old/ABC180-A.py
def solver(n: int, a: int, b: int) -> int:
    """n個のボールが入っていた箱からa個取り出し、新たにb入れた

    Args:
        n (int): [description]
        a (int): [description]
        b (int): [description]

    Returns:
        int: [description]
    """
    return max(n-a, 0)+b


def main():
    # input
    n, a, b = map(int, input().split())
    ans = solver(n, a, b)
    print(ans)


main()
